src/main/kotlin/com/sourcegraph/cody/edit/sessions/FixupSession.kt

```diff
@@ -214,14 +215,15 @@ abstract class FixupSession(
 
   private fun showWorkingGroup() {
     showLensGroup(LensGroupFactory(this).createTaskWorkingGroup())
+    documentListener.setAcceptLensGroupShown(false)
```

Comment:
we can call `showAcceptGroup` and `showWorkingGroup` multiple times in the session - e.g. when retrying or undoing

Reply:
@mkondratek that is not really true.
Then you undo session is ended.
Retry will start new edit after an undo.